### PR TITLE
Add Swift 2.0 compatibility

### PR DESCRIPTION
**Pull request by kaishin**

### CODE REVIEW

File: Source/Animator.swift [hunk 1]
```diff
@@ -32,46 +32,55 @@ class Animator {
...
   /// Loads a single frame from an image source, resizes it, then returns an `AnimatedFrame`.
   ///
-  /// :param: index The index of the GIF image source to prepare
-  /// :returns: An AnimatedFrame object
+  /// - parameter index: The index of the GIF image source to prepare
+  /// - returns: An AnimatedFrame object
   private func prepareFrame(index: Int) -> AnimatedFrame {
-    let frameDuration = CGImageSourceGIFFrameDuration(imageSource, index)
-    let frameImageRef = CGImageSourceCreateImageAtIndex(imageSource, index, nil)
+    guard let source = imageSource, let frameImageRef = CGImageSourceCreateImageAtIndex(source, index, nil) else {
```
Comment (tonyd256) [inline Source/Animator.swift]: what about a new line here after the comma

File: Source/ImageSourceHelpers.swift [hunk 2]
```diff
@@ -58,21 +58,22 @@ private func isPositive(value: Double) -> Bool {
...
-  /// :returns: A dictionary containing the GIF properties at the passed in index.
+  /// - parameter index: The index of the GIF properties to retrieve.
+  /// - returns: A dictionary containing the GIF properties at the passed in index.
   func GIFPropertiesAtIndex(index: Int) -> GIFProperties? {
-    if !isAnimatedGIF { return .None }
-
-    let imageProperties = CGImageSourceCopyPropertiesAtIndex(self, index, nil) as Dictionary
-    return imageProperties[String(kCGImagePropertyGIFDictionary)] as? GIFProperties
+    if let properties = CGImageSourceCopyPropertiesAtIndex(self, index, nil) {
+      return (properties as Dictionary)[String(kCGImagePropertyGIFDictionary)] as? GIFProperties
```
Comment (tonyd256) [inline Source/ImageSourceHelpers.swift]: can this be:
```swift
let imageProperties = CGImageSourceCopyPropertiesAtIndex(self, index, nil) as? Dictionary
return imageProperties?[String(kCGImagePropertyGIFDictionary)] as? GIFProperties
```

**Comment by tonyd256** [comment]
@kaishin I think we should change this to always have an image source. We won't ever be changing the image source in the animator since the `AnimatableImageView` creates a new one when you set a new image. I think this will simplify the `Animator` class as well.

**Comment by kaishin** [comment]
@tonyd256 Mind taking a final look here? I added specs but haven't added CI yet.

**Pull request merged**